dbms/src/Flash/Mpp/ReceivedMessageQueue.cpp

```diff
@@ -196,11 +196,12 @@ MPMCQueueResult ReceivedMessageQueue::pushAsyncGRPCPacket(size_t source_index,
     if (!received_message->containUsefulMessage())
         return MPMCQueueResult::OK;
 
+    fiu_do_on(FailPoints::random_receiver_async_msg_push_failure_failpoint, return MPMCQueueResult::CANCELLED);
```

Comment:
Maybe we can add ut like `TEST_F(ComputeServerRunner, randomFailpointForPipeline)`

Reply:
ok